homeassistant/components/venstar/climate.py

```diff
@@ -192,6 +193,7 @@ def extra_state_attributes(self):
         return {
             ATTR_FAN_STATE: self._client.fanstate,
             ATTR_HVAC_STATE: self._client.state,
+            ATTR_SCHED_PART: self._client.get_info("schedulepart")
```

Comment:
more specifically about the mismatch - why no underscore in `schedulepart` here?

Reply:
This looks correct to me actually, because that's how the API presents it.

Reply:
Yup, the comment in https://github.com/home-assistant/core/pull/84332#discussion_r1053957632 cleared things up!